core_brain/quotes.py

```
@@ -258,18 +258,28 @@ def _decide_quotes_from_mid(
-                out.append(QuoteIntent(
-                    side=side, token_id=book.get("token_id"),
-                    price=round(ba, 4), size=int(deficit), mid=mid,
-                    edge_vs_mid=mid - ba, crossed=True,
-                    reason=(f"EMERGENCY hedge: {deficit:.0f}sh short of "
-                            f"{heavy} = ${deficit_usd:.0f} vs "
-                            f"${cfg.max_naked_usd:.0f} cap, "
-                            f"{heavy} mid {heavy_mid:.3f} under avg "
-                            f"{heavy_avg:.3f} -- crossing at {ba:.3f}"),
-                ))
-                continue
+                if (heavy_avg + ba) <= pair_cap:
```

Ruff (0.16.2)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)